Formula/lua.rb

```diff
@@ -45,7 +45,8 @@ class Lua < Formula
   # sigaction provided by posix signalling power patch
   if build.with? "sigaction"
     patch do
-      url "http://lua-users.org/files/wiki_insecure/power_patches/5.2/lua-5.2.3-sig_catch.patch"
+      # url "http://lua-users.org/files/wiki_insecure/power_patches/5.2/lua-5.2.3-sig_catch.patch"
+      url "https://gist.githubusercontent.com/anonymous/99614c27bcac2aa86181fabe78c85915/raw/7abdf0234ffd38fc9094078d8128938f5e2accd4/lua-5.2.3-sig_catch.patch"
```

Comment:
From my POV it breaks up the format we do use for tappable public taps.

Pretty much everything intended to be tapped is `homebrew/homebrew-xyz` or more commonly written in terminals as `homebrew/x11` for example. I think that usage can give the impression everything that you can `brew tap homebrew/xyz` is supposed to be tapped.

I agree it's not ideal, but I think it's slightly more obvious that things named `homebrew/brew-evolution` aren't supposed to be tappable.